pkgconfig/openssl.pc.in

```diff
@@ -0,0 +1,8 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: AWS-LC
+Description: AWS-LC (OpenSSL SHIM)
+Version: @VERSION@
```

Comment:
VERSION here gets taken from CMakeLists.txt?

Reply:
Yes, so this comes out as `Version: 1.1.1`